### ISSUE TITLE
onyo edit: insufficient commit message

### ISSUE THREAD

**Issue by j-leo** [psyinfra/onyo#458]
I onyo edited a serial number and the commit does not contain the new filename.

![image](https://github.com/psyinfra/onyo/assets/56625140/3b9acecb-05d8-443f-b465-304ecace72d4)

So this step will not be discoverable in tig using the "/" search for the new name/serial
![image](https://github.com/psyinfra/onyo/assets/56625140/cde1922b-290d-4b40-aaa8-e4ec7a8eb3a8)

**Comment by j-leo** [comment]
this should at least be improved for a major change that requires the -r flag.

I do not think all changes to keys in general should be discoverable in the top view

**Comment by bpoldrack** [comment]
Actually, it is in the commit - just not in the subject line.
In your first screenshot, right hand side: The renaming is listed underneath "Inventory Operations".

Wondering about the search, though. I'm not regularly using `tig`. Is it only searching the subject lines?

**Comment by j-leo** [comment]
Yeah, I know, I did not use the right phrase. But the potential problem was not finding it with hitting `/`

**Comment by bpoldrack** [comment]
Ok, I see. Thanks for clarifying.

**Comment by j-leo** [comment]
So I'd love to see the new name in the subject line if it is such a major change

**Comment by bpoldrack** [comment]
Okay, so for me, there are several aspects to this issue:

1. I can see us changing the subject line when a rename was done to show the potential rename instead of the keys being set. However, that would be somewhat inconsistent. While it would enable this particular search, it would sabotage more generic ones (searching for modified keys).
2. I suppose this should be consistent with `edit`, leading to the question whether `edit` should come with a `-r` flag as well.
3. I don't know `tig` well, but there should be a way to search the whole commit message instead. Relying on subject only, can only ever work in very particular cases. Can someone tell whether and how to do that with `tig`?
4. Overall, a search like that seems like something that's wrong. If it really is about history, so that using `tig` makes sense, the search should be for commits involving matching files, not for anything in the commit message. If it's about finding out whether there's **currently** (not historically) an asset with that `serial` - that's a query for `onyo get` in my view. So, what really is the question you wanted answered, @j-leo ?

**Comment by j-leo** [comment]
Yes, I was trying to get the history of an asset, and `get` was not available yet and I do know only very little about it yet. 
If `get` gives me all the history there is no much sense in using `tig` instead

**Comment by bpoldrack** [comment]
Re `-r` for edit:

Verdict: Nope. edit is intentionally different and does not need that safeguard.

This aspect can be ignored when addressing/closing this issue.

**Comment by bpoldrack** [comment]
I'll close this for now.

The information is in there, searching can be accomplished by various means, and history can be looked at in different ways. Optimizing the commit message subject for a special usecase for a special feature of a special tool seems wrong.
